test/pytorch_test_base.py

```diff
@@ -312,6 +312,11 @@
     'TestIndexingXLA': {
         'test_index_put_accumulate_large_tensor_xla',  # memory limit exceeded on v2-8
     },
+
+    # test_type_promotion.py
+    'TestTypePromotionXLA': {
+        'test_bfloat16_xla',  # half support
```

Comment:
Doesn't `half` show up in the test name?

Reply:
test was doing https://github.com/pytorch/pytorch/blob/3aec1185e0643b2249984577e5f8a822b2adccde/test/test_type_promotion.py#L182 instead of doing the type decorator so we won't get a expanded name like `test_bfloat16_xla_float16`